artifact/gem5art/artifact/_artifactdb.py

```diff
@@ -179,9 +179,45 @@ def searchByLikeNameType(self, name: str, typ: str, limit: int) -> Iterable[Dict
         for d in data:
             yield d
 
+
+class MockDB(ArtifactDB):
```

Comment:
I suggest putting this in the `test_artifact.py` file.

Reply:
Done